NEON_soil_metadata/README.md

```diff
@@ -8,4 +8,5 @@ This folder includes two notebooks, in R and Python, that explore various ways o
 
 ## Python
 - [Static rendered Jupyter notebook](https://nbviewer.org/github/microbiomedata/nmdc_notebooks/blob/main/NEON_soil_metadata/python/neon_soil_metadata_visual_exploration.ipynb). _Viewing only, not editable_
-- [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/microbiomedata/nmdc_notebooks/blob/main/NEON_soil_metadata/python/neon_soil_metadata_visual_exploration.ipynb).This is the recommended way to interact with the notebook. _You need a google account to use this option_
```

Comment:
These need to be changed to the main branch, not the branch being merged in.

The NbViewer link isn't needed.

Reply:
I wasn't sure about whether we should point them to the main branch or not (I remember you mentioned it, but couldn't remember what the recommendation was :-) ). Will change it now, thanks !

Reply:
Ok, added a first cell with a "pip install" in the latest release.
@kheal : this should have solved all changes: colab worked as far as I saw (https://colab.research.google.com/github/microbiomedata/nmdc_notebooks/blob/123-python-api-to-package/NEON_soil_metadata/python/neon_soil_metadata_visual_exploration.ipynb#scrollTo=18832546-a0f8-45ad-8a6e-d700fc6c9158) , the PR title was changed, and the link in the readme points to the main branch.
Let me know if you see anything else, thanks !!